src/pages/editor/index.tsx

```diff
@@ -0,0 +1,60 @@
+import type { GetServerSidePropsContext } from 'next'
+import { NewArticle } from '@/components'
+import { NextSeo } from 'next-seo'
+import { defaultSEO } from '@/pages/_app'
+import { useEffect } from 'react'
+import { useAuth } from '@/context'
+import { useRouter } from 'next/router'
+import { parseCookies } from 'nookies'
+import styled from 'styled-components'
+
+const Wrapper = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`
+
+const Editor = () => {
+  const { status } = useAuth()
+  const router = useRouter()
+
+  useEffect(() => {
+    if (!(status === 'loggedIn')) router.replace('/unauthorized')
+  }, [status, router])
+
+  return (
+    <>
+      <NextSeo {...defaultSEO} title="Conduit - New Article" />
+      <Wrapper>
+        <NewArticle />
```

Comment:
Mover o NewArticle para ser uma page.

Reply:
What do you think about this [way](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/4f2fdca46910b916e569f906ddbdb9010e917c8e).

Reply:
I removed the Wrapper component in Editor page on this top commit, but i figured out that was important to have it, so i make another [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/1b6101acec87b701cda1184b4c4e8748dac303d8).